packages/flutter/lib/src/material/chip.dart

```diff
@@ -1821,6 +1821,13 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
     assert(debugCheckHasDirectionality(context));
     assert(debugCheckHasMaterialLocalizations(context));
 
+    final EdgeInsetsGeometry _defaultLabelPadding = EdgeInsets.lerp(
+      const EdgeInsets.symmetric(horizontal: 8.0),
+      const EdgeInsets.symmetric(horizontal: 4.0),
+      MediaQuery.of(context).textScaleFactor < 2.0
```

Comment:
You should also account for textScaleFactor being less than 1, since this will result in a negative number. It might be easier to just use `MediaQuery.of(context).textScaleFactor - 1.0` and then clamp it between 0 and 1

Reply:
done